src/main/java/com/indeed/jiraactions/ApiPageProvider.java

```diff
@@ -68,9 +67,7 @@ public long getFileWritingTime() {
     }
 
     @Override
-    public boolean hasPage() {
-        return issuesAPICaller.currentPageExist();
-    }
+    public boolean hasPage() { return issuesAPICaller.currentPageExist(); }
```

Comment:
Nit: The expanded style is preferred. Don't collapse these things to be all on one line.